dali/daliadmin/daliadmin.cpp

```diff
@@ -586,6 +588,19 @@ static void dfspart(const char *lname,IUserDescriptor *userDesc, unsigned partnu
 
 //=============================================================================
 
+static void dfsmeta(const char *filename,IUserDescriptor *userDesc, bool includeStorage)
+{
+    //This function isn't going to work on a container system because it won't have access to the storage planes
```

Comment:
but it will if executed in an esp service...
There's a PR that begins the work of moving daliadmin functionality into a new esp service.